sherpa/models/model.py

```diff
@@ -642,22 +666,40 @@ def _set_thawed_pars_maxes(self, vals):
                 p._max = v
 
     thawedparmaxes = property(_get_thawed_par_maxes, _set_thawed_pars_maxes,
-                              doc='Access to the maximum limits for the thawed parameters')
+                              doc='The maximum limits of the thawed parameters.\n\n' +
```

Comment:
While you are at it, I think it would be worthwhile to at least mention the difference between limits and hard limits. No deep discussion, but one line of text would be good either here or in the hardparmin/max or both.

Reply:
I stuck some text in the thawedparhardmins/maxes attributes and also added some text to the top-level module documentation. Let me know if it's sufficient.

Reply:
It's great.